File: week3/change/Change.cs
using System

namespace Change
{
    class Change
    {
        private static int GetChange(int m)
        {
            int[] coins = {10, 5, 1};
            int count = 0;

            for (int i = 0; m > 0; i++)
            {
                count += m / coins[i];
                m %= coins[i];
            }
            return count;
        }

        public static void Main(String[] args)
        {
            int n = int.Parse(Console.Readline());
            System.Console.WriteLine(GetChange(n));
        }
    }
}

File: week3/change/change.py
# Uses python3
import sys

def get_change(value):
    coins  = [10, 5, 1]
    count = 0
    i = 0
    while value > 0:
        count += m / coins[i]
        value %= coins[i]
        i += 1
    return value

if __name__ == '__main__':
    m = int(sys.stdin.read())
    print(get_change(m))
